build/lib/test/test.js

```diff
@@ -485,7 +485,7 @@
```

Comment:
<!--
  0 failure: 
  1 warning:  build/lib/test/te...
  
  
  DangerID: danger-id-Danger;
  File: build/lib/test/test.js;
  Line: 488;
-->

- :warning: build/lib/test/test.js line 488 – 'stripped' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)